lib/config/presets/internal/group.ts

```diff
@@ -336,6 +336,16 @@ const staticGroups = {
       },
     ],
   },
+  react: {
+    description: 'Group React and corresponding `@types` packages together.',
+    packageRules: [
+      {
+        extends: 'monorepo:react',
+        groupName: 'React packages',
```

Comment:
@rarkins do you mean this?

```suggestion
        groupName: 'react monorepo',
```

I got this from https://docs.renovatebot.com/presets-group/#groupreactmonorepo

Reply:
Yes, I think so

Reply:
And then this can be added to `group:recommended` maybe

Reply:
I think I did, could you confirm/advise?

Reply:
I think I did, could you confirm/advise?

Reply:
Yes